src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -476,8 +476,9 @@ namespace DataHeatBalance {
         Real64 minOccupants = 0.0;     // minimum occupancy (sum of NomMinNumberPeople for the space People objects, not multiplied)
         Real64 maxOccupants = 0.0;     // maximum occupancy (sum of NomMaxNumberPeople for the space People objects, not multiplied)
         bool isRemainderSpace = false; // True if this space is auto-generated "-Remainder" space
-        std::vector<ExteriorEnergyUse::ExteriorFuelUsage> otherEquipFuelTypeNums; // List of fuel types used by other equipment in this space
-        std::vector<std::string> otherEquipFuelTypeNames;                         // List of fuel types used by other equipment in this space
+        std::vector<ExteriorEnergyUse::ExteriorFuelUsage> otherEquipFuelTypeNums;         // List of fuel types used by other equipment in this space
+        std::vector<ExteriorEnergyUse::ExteriorFuelUsage> otherEquipFuelTypeNumsInclNone; // Same as above, including None
```

Comment:
I don't think `otherEquipFuelTypeNumsInclNone`is necessary. It's using `Invalid` to mark instances of "None" if I'm following correctly, which I think is the same as what's already done for `otherEquipFuelTypeNums`. We could also adding "None" to the`ExteriorEnergyUse::ExteriorFuelUsage` enum to make it more legitimate. That would require some additional validation for exterior fuel equipment, but that's probably better than using `Invalid` to mean something that's valid.

Reply:
Hmm, looking at the code again, I don't remember why I thought I had to create a new array. Let me revisit the code and see if I can add "None" to the existing `otherEquipFuelTypeNums` (and update any logic to exclude "None" where appropriate).

I agree that explicitly adding "None" to the `ExteriorEnergyUse::ExteriorFuelUsage` enum would make my code a lot cleaner, I had the same thought. But I was leery to make more extensive code changes and potentially break something. But I will look through the code a bit more to see how much effort/risk it would be.

Reply:
Okay, I think I've fully addressed this comment. The test file produces the same results. We'll see if the CI is happy.